src/SourceBuild/content/repo-projects/runtime.proj

```diff
@@ -26,6 +22,13 @@
     <BuildArgs>$(BuildArgs) /p:TargetRid=$(OverrideTargetRid)</BuildArgs>
     <BuildArgs>$(BuildArgs) /p:RuntimeOS=$(RuntimeOS)</BuildArgs>
     <BuildArgs>$(BuildArgs) /p:BaseOS=$(BaseOS)</BuildArgs>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(DotNetBuildFromSource)' == 'true'">
+    <PortableBuild Condition="'$(PortableBuild)' == ''">false</PortableBuild>
+    <BuildNonPortable>true</BuildNonPortable>
+    <BuildNonPortable Condition="'$(PortableBuild)' == 'true'">false</BuildNonPortable>
+
     <BuildArgs>$(BuildArgs) /p:PortableBuild=$(PortableBuild)</BuildArgs>
     <BuildArgs>$(BuildArgs) /p:SourceBuildNonPortable=$(BuildNonPortable)</BuildArgs>
     <BuildArgs>$(BuildArgs) /p:UsingToolMicrosoftNetCompilers=false</BuildArgs>
```

Comment:
Is this line supposed to be SB-only?

Reply:
Yes. I saw the same line in another project which talked about setting this to false to avoid prebuilds.

Reply:
The MSFT build doesn't this to false today when building runtime so we shouldn't set it here.